scripts/update/Updater.php

```diff
@@ -14,7 +14,7 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2016 (original work) Open Assessment Technologies SA;
+ * Copyright (c) 2014-2019 (original work) Open Assessment Technologies SA;
```

Comment:
Why 2014? it should be either 2016-2019 if that's the original date or simply 2019 with the latest changes.

Reply:
because original code is dating from 2014